README.md

```diff
@@ -55,6 +55,26 @@ This is the place to:
 Note: there is no need to repeat everything that is already well-documented in
 the code.
 
+If you would like to include math equations to explain parts of your code or repo,
+it is recommended to use an external renderer and insert prepared svg files here.
+For example, it's possible to insert a link to the online CodeCogs equation editor;
+
+`<img src="http://latex.codecogs.com/svg.latex?\large%20x_{ik}+x_{jk}-2x_{ik}x_{jk}">`
+
+renders the following expression.
+
+
+<img src="https://latex.codecogs.com/svg.latex?\large%20x_{ik}+x_{jk}-2x_{ik}x_{jk}">
```

Comment:
This does not show up well in dark mode. In our conversation, you had talked about a flag that would allow for this to show up in white when in dark mode. If we're going to suggest this method, it'd be good to mention.

Reply:
Yes unfortunately I have not found a satisfying way to keep readable latex in the readme and render well in both light and dark mode. It looks like GitHub is actively working on automating this as of about two months ago. https://github.community/t/support-theme-context-for-images-in-light-vs-dark-mode/147981

Stopgaps for this we recommend could easily be made obsolete, so my current opinion has changed to that it's actually not worth supporting darkmode on repos until GitHub-side support has stabilized... Let me know if that makes sense.

Reply:
Hmm, I think this could be a problem in the IDE, which defaults to dark mode. Have you taken a look at how this looks there?

Reply:
What about just including a white background in the image instead of transparent?  I think that's the way most of our current equation images are formatted.  It may not be the ideal look in dark mode, but at least it is visible.

Reply:
I have, and my conclusion was to set my IDE to light mode. I can't tell which markdown renderer is used behind the scenes in the IDE - I was wrong in my assumption that it was similar to Jupyter and it seems it has most but not all of the issues of GitHub rendering.

Reply:
I don't think we can expect users to set their IDE to a different mode.

Reply:
> What about just including a white background in the image instead of transparent? I think that's the way most of our current equation images are formatted. It may not be the ideal look in dark mode, but at least it is visible.

This is the main stopgap that I was referring to - I think it is more beneficial to have editable latex so we can easily update equations than to require inserting images pre-rendered with white backgrounds to support GitHub darkmode while it is under development unfortunately. If there is a url renderer that allows white background that would be ideal.

Reply:
Perhaps I’m missing something here, but what I’m gathering is that there is no current solution for displaying latex via a link to CodeCogs on dark mode (unless there is a url renderer w/ white background?). If that is the case, then I don’t think we should be recommending this as an option right now. This obviously could change later, but right now I’m inclined to just recommend SVG and remind users to use a white background.

Reply:
One benefit of using the HTML tag might be the ability to add background colors ourselves. I'm currently testing including the tag style="background-color:white" etc.

Reply:
Because github flavored markdown has such a [strict sanitization filter,](https://github.com/gjtorikian/html-pipeline/blob/main/lib/html/pipeline/sanitization_filter.rb#L38-L107) that was not possible; but the latex command \pagecolor{white} works just fine in the url renderer and keeps the whole insertion sufficiently unobtrusive and readable in my mind.

So now it looks alright in the IDE darkmode, though I wish there was a way to add a thin white border. In darkmode the last character in the equation is just barely frustrating to look at. I don't know a way to do this currently mainly because of the filter.